code/modules/vending/security.dm

```diff
@@ -14,14 +14,14 @@
 					/obj/item/storage/box/evidence = 6,
 					/obj/item/flashlight/seclite = 4,
 					/obj/item/restraints/legcuffs/bola/energy = 7,
+					/obj/item/clothing/head/helmet/blueshirt = 5, //SPLURT edit
+					/obj/item/clothing/under/rank/security/officer/blueshirt = 5, //SPLURT edit
+					/obj/item/clothing/suit/armor/vest/blueshirt = 5, //SPLURT edit
```

Comment:
Might wanna just do the changes in the modular file like we do with other vendors